lib/teamtailor/client.rb

```diff
@@ -68,6 +84,36 @@ def job_applications(page: 1, include: [])
       ).call
     end
 
+    def create_job_application(candidate_id:, job_id:, **args)
```

Comment:
I'm not sure about limiting the relations you can pass in like this. What do you think about changing it to something more generic (the same would go for the `#create_candidates` method)?

```rb
    def create_job_application(attrbutes:, relationships:)
      Teamtailor::Request.new(
        base_url: base_url,
        api_token: api_token,
        api_version: api_version,
        path: '/v1/job-applications',
        method: :post,
        body: {
          data: {
            type: "job-applications",
            attributes: arrtributes.transform_keys { |k| k.to_s.gsub("_", "-") },
            relationships: relationships,
          },
        }
      ).call
    end
```

This way we don't limit for passing other relations when creating the resource.

Reply:
Yeah, I think that is a good solution! I updated both methods to use attributes and relationships as arguments.